src/ontology/mondo-edit.obo

```diff
@@ -189198,6 +189198,7 @@ xref: SCTID:31470003 {source="DOID:2154"}
 is_a: MONDO:0003321 {source="MONDO:Redundant", source="OMIM:194070"} ! hereditary Wilms tumor
 is_a: MONDO:0019004 {source="DC-OMIM:194070"} ! kidney Wilms tumor
 relationship: curated_content_resource https://search.clinicalgenome.org/kb/conditions/MONDO:0008679 {source="MONDO:CLINGEN"}
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/12796 {source="OMIM:194070"} ! WT1
```

Comment:
this is a protected gene association and should be maintained https://github.com/monarch-initiative/omim/blob/main/data/protected-disease-gene.tsv#L15